outwatch/src/main/scala/outwatch/dom/helpers/EmitterBuilder.scala

```diff
@@ -1,4 +1,5 @@
-package outwatch.dom.helpers
+package outwatch
```

Comment:
Revert

Reply:
Nevermind, this is actually good because the ObserverOps are in the outwatch package.